### PR TITLE
Add Calibtool for GluonCV

### PR DESCRIPTION
**Pull request by xinyu-intel**
Pending on https://github.com/apache/incubator-mxnet/pull/15754

Add `quantize_net()` function for quantizing a gluon hybridblock and enabled in imagenet_classification and ssd models. More models are under evaluation.

UT is ready.
Docs are WIP.

@pengzhao-intel @zhreshold @wuxun-zhang

### CODE REVIEW

File: scripts/detection/ssd/README.md [hunk 1]
```diff
@@ -6,6 +6,37 @@
...
+
+## Performance
+
+model | f32 latency(ms) | s8 latency(ms) | f32 throughput(fps, BS=256) | s8 throughput(fps, BS=256) | f32 accuracy | s8 accuracy
+-- | -- | -- | -- | -- | -- | --
+ssd_300_vgg16_atrous_voc | 105.60 | 13.08 | 19.47 | 110.14 | 77.49 | 77.49
+ssd_512_vgg16_atrous_voc | 215.05 | 32.63 | 6.76 | 36.56 | 78.82 | 78.82
+ssd_512_mobilenet1.0_voc | 28.98 | 6.97 | 65.55 | 210.17 | 75.51 | 75.49
+ssd_512_resnet50_v1_voc | 52.77 | 11.75 | 28.68 | 143.61 | 80.24 | 80.23
+
```
Comment (pengzhao-intel) [inline scripts/detection/ssd/README.md]: Add FCN?

**Comment by mli** [comment]
Job PR-893-3 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-893/3/index.html 
Code coverage of this PR:  vs. Master:

File: scripts/detection/ssd/README.md (hunk 1 above)
Comment (xinyu-intel) [reply scripts/detection/ssd/README.md]: Okay, I'll apply calibtool to fcn.

Comment (wuxun-zhang) [reply scripts/detection/ssd/README.md]: Now, there is very little content displayed in `README` of segmentation part. I can work on FCN part with @xinyu-intel.

**Comment by mli** [comment]
Job PR-893-4 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-893/4/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-893-5 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-893/5/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-893-6 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-893/6/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-893-7 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-893/7/index.html 
Code coverage of this PR:  vs. Master:

**Comment by xinyu-intel** [comment]
@pengzhao-intel @zhreshold http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-893/7/build/examples_deployment/int8_inference.html

**Comment by zhreshold** [comment]
requesting @hetong007 for review as imagenet part is involved

**Comment by xinyu-intel** [comment]
Thanks for approval. Merging now:)

**Pull request merged**